src/compliance/compliance-requirements.controller.ts

```diff
@@ -32,12 +39,15 @@ export class ComplianceController {
       'List of Compliance Requirements of the Asset along with Default Trusted Claim Issuers',
     type: ComplianceRequirementsModel,
   })
+  @ApiNotFoundResponse({
+    description: 'The asset was not found',
```

Comment:
This should be "The Asset was not found". 

Asset is an entity so it gets capitalized.

Reply:
there are a few places to be updated